src/ant/src/AntennaChecker.cc

```diff
@@ -1157,31 +1159,57 @@ std::pair<bool, bool> AntennaChecker::checkWirePar(const ARinfo& AntennaRatio,
 
     if (report) {
       if (PAR_ratio != 0) {
-        if (par_violation || verbose)
+        if (par_violation || verbose) {
+          if (report_file.is_open()) {
+            report_file << "    PAR: " << par << (par_violation ? "*" : " ")
+                        << " Ratio: " << PAR_ratio << " (Area)\n";
+          }
+
           logger_->report("    PAR: {:7.2f}{} Ratio: {:7.2f} (Area)",
                           par,
                           par_violation ? "*" : " ",
                           PAR_ratio);
```

Comment:
Rather than duplicating every message it would be better to build the string once using fmt and then output it once or twice depending on the options.